WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -664,6 +666,24 @@ - (Class)classForSettingsViewController
     return [PostSettingsViewController class];
 }
 
+- (void)presentMediaCapture
+{
+    self.capturePresenter = [[WPMediaCapturePresenter alloc] initWithPresentingViewController:self];
```

Comment:
Instead of this didn't you implement an option on the WPMediaPicker to go directly to the capture screen? and avoid all this media processing code to duplicated here?

Reply:
I tried that here, but then you have to:

* Retain an instance of a media picker (it's different on Aztec, because a picker is already visible)
* Present the media picker and then immediately present the capture UI on top (which could be confusing to a user)
* Passing the image back through to the picker and then automatically selecting and adding it is a bit awkward.
* Dismiss the capture UI and the picker UI from underneath.

For this situation, being able to just present the capture UI directly actually feels like a nicer solution to me.